.ci/Jenkinsfile

```diff
@@ -18,7 +18,23 @@
 @Library('apm@master') _
 
 pipeline {
-  agent { label 'linux && immutable' }
+  agent {
+    kubernetes {
+      yaml '''
+apiVersion: v1
+kind: Pod
+spec:
+  containers:
+  - name: shell
+    image: ubuntu
+    command:
+    - sleep
+    args:
+    - infinity
+  '''
+      defaultContainer 'shell'
+    }
```

Comment:
🤔 I think we can replace this definition with a step like it is possible to make with whole pipelines. It is an idea to test in the k8s project. Something like this:

```
pipeline {
  agent {  ubuntuPod() }

...
```

ubuntuPod.groovy
```
call(){
    kubernetes {
      yaml '''
apiVersion: v1
kind: Pod
spec:
  containers:
  - name: shell
    image: ubuntu
    command:
    - sleep
    args:
    - infinity
  '''
      defaultContainer 'shell'
    }
```

Reply:
Using `yamlFile`